docs/logs/faq.md

```diff
@@ -576,12 +570,11 @@ formats.
 
 **What happens if I try to send data with @timestamp older than my retention time?**
 
-You can send old logs.  However, keep in mind your logging app's
-retention settings.  For example, if your retention is set to 7 days and
+You can send old logs because our retention machanism works on received timestampl, not @timestamp.
+For example, if your retention is set to 7 days and
 you send logs that are 10 days old (i.e. they contain a timestamp from
-10 days ago), they will get accepted and processed, but will get removed
-within the next 24
-hours.
+10 days ago), they will get accepted and processed, but you when you set time range to select those logs,
+you might get warning that you selected time range that is outside of your retention.
```

Comment:
you might get warning that you selected time range that is outside of your retention.
=>
you might get a warning that your selected time range is outside of your retention.